### PR TITLE
Zib-HealthcareProvider

### PR DESCRIPTION
**Pull request by AmyKoo1**
-

### CODE REVIEW

File: known-issues.yml [hunk 1]
```diff
@@ -4,9 +4,36 @@
...
 # The following issues are not "real" deviations stemming from design choices, but rather problems that pop up due to
 # shortcomings in tooling, the used terminology server, etc.
 # These might occur in one or more profiles, or in none if they have been fixed in the meantime. 
 
 issues should occur: false
+
+zib-HealthcareProvider:
+  ignored issues:
+    Location.address:
+      - message: "Constraint failed: sd-pg-02: 'If mapping.map exists and the mapping is not implicit, short should exist'"
```
Comment (ArdonToonstra) [inline known-issues.yml:36]: Perhaps we can add the short and alias from the NL-CM:17.2.5 concept and remove this?

**Comment by ArdonToonstra** [comment]
We still should add the release-notes if nothing has changed. In this case you would add one line indicating that there are no changes. However, the release notes should contain the changes for ZIB-1927 and ZIB-1559 and potentially ZIB-1665.

**Review by ArdonToonstra — DISMISSED** [review]

File: known-issues.yml (hunk 1 above)
Comment (AmyKoo1) [reply known-issues.yml]: I kept this, otherwise we would have to provide the same alias and short already given by the base profile

Comment (ArdonToonstra) [reply known-issues.yml:84]: Yes, you are right. I will resolve this conversation.

**Review by LuudSlagter — APPROVED** [review]

File: examples/nl-core-HealthcareProvider-Organization-01.xml [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+      <display value="Cardiologie"/>
+    </coding>
+  </type>
+  <type>
+    <coding>
+      <system value="http://nictiz.nl/fhir/NamingSystem/organization-type"/>
+      <code value="V4"/>
+      <display value="Ziekenhuis"/>
+    </coding>
+  </type>
```
Comment (ArdonToonstra) [inline examples/nl-core-HealthcareProvider-Organization-01.xml:59]: I am missing a name while this is in the ada instance.

**Review by ArdonToonstra — CHANGES_REQUESTED** [review]

**Review by ArdonToonstra — APPROVED** [review]
LGTM now!

**Pull request merged**